tests/phpunit/tests/Performances/test-Post_Translations_Group.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace WP_Syntex\Polylang\Tests\Integration\Performances;
+
+use PLL_UnitTestCase;
+use PLL_UnitTest_Factory;
+class Post_Translations_Group_Test extends PLL_UnitTestCase {
+
+	/**
+	 * @param PLL_UnitTest_Factory $factory
+	 * @return void
+	 */
+	public static function pllSetUpBeforeClass( PLL_UnitTest_Factory $factory ) {
+		parent::pllSetUpBeforeClass( $factory );
+
+		$factory->language->create_many( 2 );
+	}
+
+	/**
+	 * Checks that updating a post translations group is done only once when we unlink all translations.
+	 */
+	public function test_should_not_update_translations_group_when_removing_all_translations() {
```

Comment:
OK for the fix. However I would not insert this test in a separate folder.
1. This is not a performance test (because the performance is not measured)
2. It would be separated from related tests and future us would forget its existence.

So I suggest that you move this test to the existing related class. You'll see that you can also simplify it, i.e; remove some assertions as they are already checked in another test. See https://github.com/polylang/polylang/blob/master/tests/phpunit/tests/test-translated-post.php#L65-L69 (this one could be completed as there are only 6/9 assertions).

Reply:
Oops... Sorry @manooweb for pointed in the wrong direction.